docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md

```diff
@@ -110,7 +116,10 @@ Use the following steps to deploy an Azure cluster.
    - Once the `fullyPrivateAddressing` parameter is set for your cluster, you cannot change its value. Changing the
      parameter value will result in errors until you return the value to its original configuration.
 
-   :::
+   Toggle the **Private API Server** option to enable the use of a Private API Server load balancer and specify the
```

Comment:
```suggestion
   Toggle the **Private API Server LB** option to enable the use of a Private API Server load balancer and specify the
```
Also, I would add this paragraph to step number 12 of the guide (Cluster config) :)